user_manual/ios_app.rst

```diff
@@ -67,6 +87,52 @@ If the file is filled white, then it is available offline.
 Click the button to make the file available, or not available offline.
 
 Sharing Files
+~~~~~~~~~~~~~
+
+The triangular Share button opens a share dialog, allowing you to start
```

Comment:
allowing you to share by link or with other ownCloud users
This option is different than the sharing files section previously described